tests/functional/z_cluster/nodes/test_node_replacement_proactive.py

```
@@ -345,25 +353,33 @@ class TestNodeReplacementTwice(ManageTest):
 
     @pytest.fixture(autouse=True)
     def teardown(self, request):
```

Ruff (0.15.15)

[warning] Unused method argument: `request`

(ARG002)